components/blog-article.tpl

```diff
@@ -12,7 +12,15 @@
   </header>
 
   <div class="article__content" data-search-indexing-allowed="true">
-    <div class="content-area content-area--article-excerpt content-area--overflowed-images"{% unless list_item %} {{ edy_intro_edit_text }}{% endunless %}>{% if list_item %}{{ article.excerpt }}{% else %}{% editable article.excerpt %}{% endif %}</div>
+    <div class="content-area content-area--article-excerpt content-area--overflowed-images"{% unless list_item %} {{ edy_intro_edit_text }}{% endunless %}>
+      {% if list_item %}{{ article.excerpt }}{% else %}{% editable article.excerpt %}{% endif %}
+    </div>
+
+    <div class="content-area">
+      {% if list_item %}
+        <a class="readmore-link" href="{{ article.url }}">{{ "read_more" | lc }}</a>
```

Comment:
This template is written using [BEM methodology](https://en.bem.info/methodology/quick-start/) and this addition should follow its rules.

In that case you have created a [block](https://en.bem.info/methodology/quick-start/#block) named **readmore-link** which is not correct. It should be an [element](https://en.bem.info/methodology/quick-start/#element) which is a child of the **article** block. Sho the element class should be something like `.article__read-more`.